src/app/wallets/get-balance-for-wallet.ts

```diff
@@ -10,23 +11,31 @@ export const getBalanceForWallet = async ({
   walletId: WalletId
   lock?: DistributedLock
   logger: Logger
-}): Promise<Satoshis | ApplicationError> => {
-  const [, updatePaymentsResult] = await Promise.all([
-    Wallets.updatePendingInvoices({
-      walletId: walletId,
-      lock,
-      logger,
-    }),
-    Wallets.updatePendingPayments({
-      walletId: walletId,
-      lock,
-      logger,
-    }),
-  ])
-  if (updatePaymentsResult instanceof Error) throw updatePaymentsResult
+}): Promise<Satoshis | ApplicationError> =>
+  asyncRunInSpan(
+    "app.getBalanceForWallet",
+    {
+      [SemanticAttributes.CODE_FUNCTION]: "getBalanceForWallet",
+      "enduser.walletId": walletId,
```

Comment:
This should be defined globally like [`ENDUSER_ALIAS`](https://github.com/GaloyMoney/galoy/blob/main/src/services/tracing.ts#L189).
Also it is semantically wrong as the enduser doesn't have a wallet - an account has wallets. Not a blocker as this adds value.